<!DOCTYPE html>
<html>
<head>
<title>ATR.md</title>
<meta http-equiv="Content-type" content="text/html;charset=UTF-8">

<style>
/* https://github.com/microsoft/vscode/blob/master/extensions/markdown-language-features/media/markdown.css */
/*---------------------------------------------------------------------------------------------
 *  Copyright (c) Microsoft Corporation. All rights reserved.
 *  Licensed under the MIT License. See License.txt in the project root for license information.
 *--------------------------------------------------------------------------------------------*/

body {
	font-family: var(--vscode-markdown-font-family, -apple-system, BlinkMacSystemFont, "Segoe WPC", "Segoe UI", "Ubuntu", "Droid Sans", sans-serif);
	font-size: var(--vscode-markdown-font-size, 14px);
	padding: 0 26px;
	line-height: var(--vscode-markdown-line-height, 22px);
	word-wrap: break-word;
}

#code-csp-warning {
	position: fixed;
	top: 0;
	right: 0;
	color: white;
	margin: 16px;
	text-align: center;
	font-size: 12px;
	font-family: sans-serif;
	background-color:#444444;
	cursor: pointer;
	padding: 6px;
	box-shadow: 1px 1px 1px rgba(0,0,0,.25);
}

#code-csp-warning:hover {
	text-decoration: none;
	background-color:#007acc;
	box-shadow: 2px 2px 2px rgba(0,0,0,.25);
}

body.scrollBeyondLastLine {
	margin-bottom: calc(100vh - 22px);
}

body.showEditorSelection .code-line {
	position: relative;
}

body.showEditorSelection .code-active-line:before,
body.showEditorSelection .code-line:hover:before {
	content: "";
	display: block;
	position: absolute;
	top: 0;
	left: -12px;
	height: 100%;
}

body.showEditorSelection li.code-active-line:before,
body.showEditorSelection li.code-line:hover:before {
	left: -30px;
}

.vscode-light.showEditorSelection .code-active-line:before {
	border-left: 3px solid rgba(0, 0, 0, 0.15);
}

.vscode-light.showEditorSelection .code-line:hover:before {
	border-left: 3px solid rgba(0, 0, 0, 0.40);
}

.vscode-light.showEditorSelection .code-line .code-line:hover:before {
	border-left: none;
}

.vscode-dark.showEditorSelection .code-active-line:before {
	border-left: 3px solid rgba(255, 255, 255, 0.4);
}

.vscode-dark.showEditorSelection .code-line:hover:before {
	border-left: 3px solid rgba(255, 255, 255, 0.60);
}

.vscode-dark.showEditorSelection .code-line .code-line:hover:before {
	border-left: none;
}

.vscode-high-contrast.showEditorSelection .code-active-line:before {
	border-left: 3px solid rgba(255, 160, 0, 0.7);
}

.vscode-high-contrast.showEditorSelection .code-line:hover:before {
	border-left: 3px solid rgba(255, 160, 0, 1);
}

.vscode-high-contrast.showEditorSelection .code-line .code-line:hover:before {
	border-left: none;
}

img {
	max-width: 100%;
	max-height: 100%;
}

a {
	text-decoration: none;
}

a:hover {
	text-decoration: underline;
}

a:focus,
input:focus,
select:focus,
textarea:focus {
	outline: 1px solid -webkit-focus-ring-color;
	outline-offset: -1px;
}

hr {
	border: 0;
	height: 2px;
	border-bottom: 2px solid;
}

h1 {
	padding-bottom: 0.3em;
	line-height: 1.2;
	border-bottom-width: 1px;
	border-bottom-style: solid;
}

h1, h2, h3 {
	font-weight: normal;
}

table {
	border-collapse: collapse;
}

table > thead > tr > th {
	text-align: left;
	border-bottom: 1px solid;
}

table > thead > tr > th,
table > thead > tr > td,
table > tbody > tr > th,
table > tbody > tr > td {
	padding: 5px 10px;
}

table > tbody > tr + tr > td {
	border-top: 1px solid;
}

blockquote {
	margin: 0 7px 0 5px;
	padding: 0 16px 0 10px;
	border-left-width: 5px;
	border-left-style: solid;
}

code {
	font-family: Menlo, Monaco, Consolas, "Droid Sans Mono", "Courier New", monospace, "Droid Sans Fallback";
	font-size: 1em;
	line-height: 1.357em;
}

body.wordWrap pre {
	white-space: pre-wrap;
}

pre:not(.hljs),
pre.hljs code > div {
	padding: 16px;
	border-radius: 3px;
	overflow: auto;
}

pre code {
	color: var(--vscode-editor-foreground);
	tab-size: 4;
}

/** Theming */

.vscode-light pre {
	background-color: rgba(220, 220, 220, 0.4);
}

.vscode-dark pre {
	background-color: rgba(10, 10, 10, 0.4);
}

.vscode-high-contrast pre {
	background-color: rgb(0, 0, 0);
}

.vscode-high-contrast h1 {
	border-color: rgb(0, 0, 0);
}

.vscode-light table > thead > tr > th {
	border-color: rgba(0, 0, 0, 0.69);
}

.vscode-dark table > thead > tr > th {
	border-color: rgba(255, 255, 255, 0.69);
}

.vscode-light h1,
.vscode-light hr,
.vscode-light table > tbody > tr + tr > td {
	border-color: rgba(0, 0, 0, 0.18);
}

.vscode-dark h1,
.vscode-dark hr,
.vscode-dark table > tbody > tr + tr > td {
	border-color: rgba(255, 255, 255, 0.18);
}

</style>

<style>
/* Tomorrow Theme */
/* http://jmblog.github.com/color-themes-for-google-code-highlightjs */
/* Original theme - https://github.com/chriskempson/tomorrow-theme */

/* Tomorrow Comment */
.hljs-comment,
.hljs-quote {
	color: #8e908c;
}

/* Tomorrow Red */
.hljs-variable,
.hljs-template-variable,
.hljs-tag,
.hljs-name,
.hljs-selector-id,
.hljs-selector-class,
.hljs-regexp,
.hljs-deletion {
	color: #c82829;
}

/* Tomorrow Orange */
.hljs-number,
.hljs-built_in,
.hljs-builtin-name,
.hljs-literal,
.hljs-type,
.hljs-params,
.hljs-meta,
.hljs-link {
	color: #f5871f;
}

/* Tomorrow Yellow */
.hljs-attribute {
	color: #eab700;
}

/* Tomorrow Green */
.hljs-string,
.hljs-symbol,
.hljs-bullet,
.hljs-addition {
	color: #718c00;
}

/* Tomorrow Blue */
.hljs-title,
.hljs-section {
	color: #4271ae;
}

/* Tomorrow Purple */
.hljs-keyword,
.hljs-selector-tag {
	color: #8959a8;
}

.hljs {
	display: block;
	overflow-x: auto;
	color: #4d4d4c;
	padding: 0.5em;
}

.hljs-emphasis {
	font-style: italic;
}

.hljs-strong {
	font-weight: bold;
}
</style>

<style>
/*
 * Markdown PDF CSS
 */

 body {
	font-family: -apple-system, BlinkMacSystemFont, "Segoe WPC", "Segoe UI", "Ubuntu", "Droid Sans", sans-serif, "Meiryo";
	padding: 0 12px;
}

pre {
	background-color: #f8f8f8;
	border: 1px solid #cccccc;
	border-radius: 3px;
	overflow-x: auto;
	white-space: pre-wrap;
	overflow-wrap: break-word;
}

pre:not(.hljs) {
	padding: 23px;
	line-height: 19px;
}

blockquote {
	background: rgba(127, 127, 127, 0.1);
	border-color: rgba(0, 122, 204, 0.5);
}

.emoji {
	height: 1.4em;
}

code {
	font-size: 14px;
	line-height: 19px;
}

/* for inline code */
:not(pre):not(.hljs) > code {
	color: #C9AE75; /* Change the old color so it seems less like an error */
	font-size: inherit;
}

/* Page Break : use <div class="page"/> to insert page break
-------------------------------------------------------- */
.page {
	page-break-after: always;
}

</style>

<script src="https://unpkg.com/mermaid/dist/mermaid.min.js"></script>
</head>
<body>
  <script>
    mermaid.initialize({
      startOnLoad: true,
      theme: document.body.classList.contains('vscode-dark') || document.body.classList.contains('vscode-high-contrast')
          ? 'dark'
          : 'default'
    });
  </script>
<h1 id="atr">ATR</h1>
<h2 id="history">History</h2>
<p>Mutations in this gene were first described in DLBCL in 2017 by Reddy et al.<sup>1</sup> Subsequent exome and genome-wide studies of DLBCL did not reproduce this observation.</p>
<pre><code class="language-mermaid"><div class="mermaid">%%{init: { 'logLevel': 'debug', 'theme': 'dark' } }%%
timeline
    title Publication timing
      2017-10-10 : Reddy : DLBCL
</div></code></pre>
<h2 id="relevance-tier-by-entity">Relevance tier by entity</h2>
<table>
<thead>
<tr>
<th style="text-align:center">Entity</th>
<th style="text-align:center">Tier</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center"><img src="file:///Users/rmorin/git/LLMPP.wiki/images/icons/DLBCL_tier2.png" alt="DLBCL"></td>
<td style="text-align:center">2</td>
<td>relevance in DLBCL not firmly established[@reddyGeneticFunctionalDrivers2017]</td>
</tr>
</tbody>
</table>
<h2 id="mutation-incidence-in-large-patient-cohorts-gambl-reanalysis">Mutation incidence in large patient cohorts (GAMBL reanalysis)</h2>
<table>
<thead>
<tr>
<th style="text-align:center">Entity</th>
<th style="text-align:center">source</th>
<th style="text-align:center">frequency (%)</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center">DLBCL</td>
<td style="text-align:center">GAMBL genomes</td>
<td style="text-align:center">2.68</td>
</tr>
<tr>
<td style="text-align:center">DLBCL</td>
<td style="text-align:center">Schmitz cohort</td>
<td style="text-align:center">4.89</td>
</tr>
<tr>
<td style="text-align:center">DLBCL</td>
<td style="text-align:center">Reddy cohort</td>
<td style="text-align:center">2.80</td>
</tr>
<tr>
<td style="text-align:center">DLBCL</td>
<td style="text-align:center">Chapuy cohort</td>
<td style="text-align:center">3.85</td>
</tr>
</tbody>
</table>
<h2 id="mutation-pattern-and-selective-pressure-estimates">Mutation pattern and selective pressure estimates</h2>
<table>
<thead>
<tr>
<th style="text-align:center">Entity</th>
<th style="text-align:center">aSHM</th>
<th style="text-align:center">Significant selection</th>
<th style="text-align:center">dN/dS (missense)</th>
<th style="text-align:center">dN/dS (nonsense)</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center">BL</td>
<td style="text-align:center">No</td>
<td style="text-align:center">No</td>
<td style="text-align:center">0.459</td>
<td style="text-align:center">0.000</td>
</tr>
<tr>
<td style="text-align:center">DLBCL</td>
<td style="text-align:center">No</td>
<td style="text-align:center">No</td>
<td style="text-align:center">1.146</td>
<td style="text-align:center">3.261</td>
</tr>
<tr>
<td style="text-align:center">FL</td>
<td style="text-align:center">No</td>
<td style="text-align:center">No</td>
<td style="text-align:center">0.000</td>
<td style="text-align:center">0.000</td>
</tr>
</tbody>
</table>
<p>View coding variants in ProteinPaint <a href="https://morinlab.github.io/LLMPP/GAMBL/ATR_protein.html">hg19</a>  or <a href="https://morinlab.github.io/LLMPP/GAMBL/ATR_protein_hg38.html">hg38</a></p>
<p><img src="file:///Users/rmorin/git/LLMPP.wiki/images/proteinpaint/ATR_NM_001184.svg" alt=""></p>
<p>View all variants in GenomePaint <a href="https://morinlab.github.io/LLMPP/GAMBL/ATR.html">hg19</a>  or <a href="https://morinlab.github.io/LLMPP/GAMBL/ATR_hg38.html">hg38</a></p>
<p><img src="file:///Users/rmorin/git/LLMPP.wiki/images/proteinpaint/ATR.svg" alt=""></p>
<h2 id="atr-expression">ATR Expression</h2>
<p><img src="file:///Users/rmorin/git/LLMPP.wiki/images/gene_expression/ATR_by_pathology.svg" alt=""></p>
<h2 id="representative-mutations">Representative Mutations</h2>
<p><img src="file:///Users/rmorin/git/LLMPP.wiki/primary/Reddy_ATR_1.svg" alt="">
<strong>Rating</strong>
★ ★ ★ ★ ★</p>
<p><img src="file:///Users/rmorin/git/LLMPP.wiki/primary/Reddy_ATR_2.svg" alt="">
<strong>Rating</strong>
★ ★ ☆ ☆ ☆</p>
<h2 id="all-mutations">All Mutations</h2>
<p><a href="https://www.bcgsc.ca/downloads/morinlab/GAMBL/Reddy/igv_reports/Reddy_2121T.html">Reddy_2121T</a>
<a href="https://www.bcgsc.ca/downloads/morinlab/GAMBL/Reddy/igv_reports/Reddy_2150T.html">Reddy_2150T</a>
<a href="https://www.bcgsc.ca/downloads/morinlab/GAMBL/Reddy/igv_reports/Reddy_2163T.html">Reddy_2163T</a>
<a href="https://www.bcgsc.ca/downloads/morinlab/GAMBL/Reddy/igv_reports/Reddy_2192T.html">Reddy_2192T</a>
<a href="https://www.bcgsc.ca/downloads/morinlab/GAMBL/Reddy/igv_reports/Reddy_2195T.html">Reddy_2195T</a>
<a href="https://www.bcgsc.ca/downloads/morinlab/GAMBL/Reddy/igv_reports/Reddy_2206T.html">Reddy_2206T</a>
<a href="https://www.bcgsc.ca/downloads/morinlab/GAMBL/Reddy/igv_reports/Reddy_2236T.html">Reddy_2236T</a>
<a href="https://www.bcgsc.ca/downloads/morinlab/GAMBL/Reddy/igv_reports/Reddy_2475T.html">Reddy_2475T</a>
<a href="https://www.bcgsc.ca/downloads/morinlab/GAMBL/Reddy/igv_reports/Reddy_2476T.html">Reddy_2476T</a>
<a href="https://www.bcgsc.ca/downloads/morinlab/GAMBL/Reddy/igv_reports/Reddy_2585T.html">Reddy_2585T</a>
<a href="https://www.bcgsc.ca/downloads/morinlab/GAMBL/Reddy/igv_reports/Reddy_2594T.html">Reddy_2594T</a>
<a href="https://www.bcgsc.ca/downloads/morinlab/GAMBL/Reddy/igv_reports/Reddy_2758T.html">Reddy_2758T</a>
<a href="https://www.bcgsc.ca/downloads/morinlab/GAMBL/Reddy/igv_reports/Reddy_2768T.html">Reddy_2768T</a>
<a href="https://www.bcgsc.ca/downloads/morinlab/GAMBL/Reddy/igv_reports/Reddy_2775T.html">Reddy_2775T</a>
<a href="https://www.bcgsc.ca/downloads/morinlab/GAMBL/Reddy/igv_reports/Reddy_2779T.html">Reddy_2779T</a>
<a href="https://www.bcgsc.ca/downloads/morinlab/GAMBL/Reddy/igv_reports/Reddy_2805T.html">Reddy_2805T</a>
<a href="https://www.bcgsc.ca/downloads/morinlab/GAMBL/Reddy/igv_reports/Reddy_2828T.html">Reddy_2828T</a>
<a href="https://www.bcgsc.ca/downloads/morinlab/GAMBL/Reddy/igv_reports/Reddy_2838T.html">Reddy_2838T</a>
<a href="https://www.bcgsc.ca/downloads/morinlab/GAMBL/Reddy/igv_reports/Reddy_2898T.html">Reddy_2898T</a>
<a href="https://www.bcgsc.ca/downloads/morinlab/GAMBL/Reddy/igv_reports/Reddy_2933T.html">Reddy_2933T</a>
<a href="https://www.bcgsc.ca/downloads/morinlab/GAMBL/Reddy/igv_reports/Reddy_2947T.html">Reddy_2947T</a>
<a href="https://www.bcgsc.ca/downloads/morinlab/GAMBL/Reddy/igv_reports/Reddy_3384T.html">Reddy_3384T</a>
<a href="https://www.bcgsc.ca/downloads/morinlab/GAMBL/Reddy/igv_reports/Reddy_3385T.html">Reddy_3385T</a>
<a href="https://www.bcgsc.ca/downloads/morinlab/GAMBL/Reddy/igv_reports/Reddy_3405T.html">Reddy_3405T</a>
<a href="https://www.bcgsc.ca/downloads/morinlab/GAMBL/Reddy/igv_reports/Reddy_3489T.html">Reddy_3489T</a>
<a href="https://www.bcgsc.ca/downloads/morinlab/GAMBL/Reddy/igv_reports/Reddy_3502T.html">Reddy_3502T</a>
<a href="https://www.bcgsc.ca/downloads/morinlab/GAMBL/Reddy/igv_reports/Reddy_3520T.html">Reddy_3520T</a>
<a href="https://www.bcgsc.ca/downloads/morinlab/GAMBL/Reddy/igv_reports/Reddy_3548T.html">Reddy_3548T</a>
<a href="https://www.bcgsc.ca/downloads/morinlab/GAMBL/Reddy/igv_reports/Reddy_3549T.html">Reddy_3549T</a>
<a href="https://www.bcgsc.ca/downloads/morinlab/GAMBL/Reddy/igv_reports/Reddy_3557T.html">Reddy_3557T</a>
<a href="https://www.bcgsc.ca/downloads/morinlab/GAMBL/Reddy/igv_reports/Reddy_3561T.html">Reddy_3561T</a>
<a href="https://www.bcgsc.ca/downloads/morinlab/GAMBL/Reddy/igv_reports/Reddy_3562T.html">Reddy_3562T</a>
<a href="https://www.bcgsc.ca/downloads/morinlab/GAMBL/Reddy/igv_reports/Reddy_3594T.html">Reddy_3594T</a>
<a href="https://www.bcgsc.ca/downloads/morinlab/GAMBL/Reddy/igv_reports/Reddy_3622T.html">Reddy_3622T</a>
<a href="https://www.bcgsc.ca/downloads/morinlab/GAMBL/Reddy/igv_reports/Reddy_3653T.html">Reddy_3653T</a>
<a href="https://www.bcgsc.ca/downloads/morinlab/GAMBL/Reddy/igv_reports/Reddy_3662T.html">Reddy_3662T</a>
<a href="https://www.bcgsc.ca/downloads/morinlab/GAMBL/Reddy/igv_reports/Reddy_3779T.html">Reddy_3779T</a>
<a href="https://www.bcgsc.ca/downloads/morinlab/GAMBL/Reddy/igv_reports/Reddy_3810T.html">Reddy_3810T</a>
<a href="https://www.bcgsc.ca/downloads/morinlab/GAMBL/Reddy/igv_reports/Reddy_3812T.html">Reddy_3812T</a>
<a href="https://www.bcgsc.ca/downloads/morinlab/GAMBL/Reddy/igv_reports/Reddy_3820T.html">Reddy_3820T</a>
<a href="https://www.bcgsc.ca/downloads/morinlab/GAMBL/Reddy/igv_reports/Reddy_3822T.html">Reddy_3822T</a>
<a href="https://www.bcgsc.ca/downloads/morinlab/GAMBL/Reddy/igv_reports/Reddy_3843T.html">Reddy_3843T</a>
<a href="https://www.bcgsc.ca/downloads/morinlab/GAMBL/Reddy/igv_reports/Reddy_3856T.html">Reddy_3856T</a>
<a href="https://www.bcgsc.ca/downloads/morinlab/GAMBL/Reddy/igv_reports/Reddy_3879T.html">Reddy_3879T</a>
<a href="https://www.bcgsc.ca/downloads/morinlab/GAMBL/Reddy/igv_reports/Reddy_3992T.html">Reddy_3992T</a>
<a href="https://www.bcgsc.ca/downloads/morinlab/GAMBL/Reddy/igv_reports/Reddy_705T.html">Reddy_705T</a></p>
<h2 id="references">References</h2>
<!-- ORIGIN: reddyGeneticFunctionalDrivers2017 -->
<!-- DLBCL: reddyGeneticFunctionalDrivers2017 -->

</body>
</html>
