# DLBCL genes

## Tier 1 DLBCL genes

|   Gene    | Tier |               Relevant references                |
|:---------:|:----:|:------------------------------------------------:|
|   ACTB    |  1   |                (Lohr et al. 2012)                |
|   ACTG1   |  1   |             (Hübschmann et al. 2021)             |
|  ARID1A   |  1   |     (Zhang et al. 2013; Krysiak et al. 2017)     |
|    ATM    |  1   |               (Reddy et al. 2017)                |
|    B2M    |  1   |               (Morin et al. 2011)                |
|   BCL10   |  1   | (Morin et al. 2011; Russler-Germain et al. 2023) |
|  BCL11A   |  1   |                                                  |
|   BCL2    |  1   |     (Tanaka et al. 1992; Morin et al. 2011)      |
|   BCL6    |  1   |               (Morin et al. 2011)                |
|   BCL7A   |  1   |    (Arthur et al. 2018; Krysiak et al. 2017)     |
|   BCOR    |  1   |                                                  |
|    BCR    |  1   |                                                  |
|   BIRC6   |  1   |               (Reddy et al. 2017)                |
|   BRAF    |  1   |               (Tiacci et al. 2011)               |
|   BTG1    |  1   |               (Morin et al. 2011)                |
|   BTG2    |  1   |               (Morin et al. 2011)                |
|    BTK    |  1   |  (Albuquerque et al. 2017; Krysiak et al. 2017)  |
|  CARD11   |  1   |      (Lenz et al. 2008; Morin et al. 2011)       |
|   CASP8   |  1   |               (Reddy et al. 2017)                |
|   CCND3   |  1   |               (Morin et al. 2011)                |
|   CD274   |  1   |               (Morin et al. 2011)                |
|   CD36    |  1   |    (Laura Pasqualucci, Trifonov, et al. 2011)    |
|   CD58    |  1   |               (Morin et al. 2011)                |
|   CD70    |  1   | (Morin et al. 2011; Russler-Germain et al. 2023) |
|   CD79B   |  1   |               (Morin et al. 2011)                |
|   CD83    |  1   | (Morin et al. 2013; Russler-Germain et al. 2023) |
|  CDKN2A   |  1   |               (Morin et al. 2013)                |
|   CIITA   |  1   |               (Morin et al. 2011)                |
|  CREBBP   |  1   | (Laura Pasqualucci, Dominguez-Sola, et al. 2011) |
|   CXCR4   |  1   | (Khodabakhshi et al. 2012; Krysiak et al. 2017)  |
|   CXCR5   |  1   |              (Schmitz et al. 2018)               |
|  DAZAP1   |  1   |           (Pararajalingam et al. 2020)           |
|   DDX3X   |  1   |              (Schmitz et al. 2018)               |
|   DTX1    |  1   |              (Schmitz et al. 2018)               |
|   DUSP2   |  1   |               (Morin et al. 2013)                |
|   EBF1    |  1   |               (Bohle et al. 2013)                |
|  EEF1A1   |  1   |             (Hübschmann et al. 2021)             |
|   EP300   |  1   | (Laura Pasqualucci, Dominguez-Sola, et al. 2011) |
|   ETS1    |  1   |               (Morin et al. 2011)                |
|   ETV6    |  1   |               (Arthur et al. 2018)               |
|   EZH2    |  1   |               (Morin et al. 2010)                |
|    FAS    |  1   |               (Scholl et al. 2007)               |
|  FBXO11   |  1   |             (Hübschmann et al. 2021)             |
|   FBXW7   |  1   |               (Zhang et al. 2013)                |
|   FOXC1   |  1   |              (Schmitz et al. 2018)               |
|   FOXO1   |  1   |               (Morin et al. 2011)                |
|   GNA13   |  1   |               (Morin et al. 2011)                |
|   GNAI2   |  1   |               (Morin et al. 2013)                |
|   GRB2    |  1   |    (Laura Pasqualucci, Trifonov, et al. 2011)    |
|   GRHPR   |  1   |               (Arthur et al. 2018)               |
| HIST1H1B  |  1   |    (Chapuy et al. 2018; Krysiak et al. 2017)     |
| HIST1H1C  |  1   |               (Morin et al. 2011)                |
| HIST1H1D  |  1   |     (Morin et al. 2013; Krysiak et al. 2017)     |
| HIST1H1E  |  1   |     (Morin et al. 2013; Krysiak et al. 2017)     |
| HIST1H2AC |  1   |     (Morin et al. 2013; Krysiak et al. 2017)     |
| HIST1H2AM |  1   |              (Krysiak et al. 2017)               |
| HIST1H2BC |  1   |     (Reddy et al. 2017; Krysiak et al. 2017)     |
| HIST1H2BK |  1   |                                                  |
| HIST1H3B  |  1   |               (Zhang et al. 2013)                |
| HIST2H2BE |  1   |              (Schmitz et al. 2018)               |
|   HLA-A   |  1   |                                                  |
|   HLA-B   |  1   |                                                  |
|   HLA-C   |  1   |                                                  |
|  HLA-DMA  |  1   |                                                  |
|  HLA-DMB  |  1   |                                                  |
| HLA-DQA1  |  1   |             (Hübschmann et al. 2021)             |
|  HNRNPD   |  1   |                                                  |
|  HNRNPH1  |  1   |           (Pararajalingam et al. 2020)           |
|  HNRNPU   |  1   |               (Reddy et al. 2017)                |
|   HVCN1   |  1   |              (Krysiak et al. 2017)               |
|   IKZF3   |  1   |               (Morin et al. 2013)                |
|   IL16    |  1   |                                                  |
|   IL4R    |  1   |                (Duns et al. 2021)                |
|   IRF4    |  1   |               (Morin et al. 2011)                |
|   IRF8    |  1   |               (Morin et al. 2011)                |
|   ITPKB   |  1   |              (Schmitz et al. 2018)               |
|   JUNB    |  1   |               (Reddy et al. 2017)                |
|   KLF2    |  1   |    (Laura Pasqualucci, Trifonov, et al. 2011)    |
|  KLHL14   |  1   |               (Zhang et al. 2013)                |
|   KLHL6   |  1   |               (Morin et al. 2011)                |
|   KMT2C   |  1   |               (Zhang et al. 2013)                |
|   KMT2D   |  1   |               (Morin et al. 2011)                |
|   KRAS    |  1   |                (Lohr et al. 2012)                |
|   LCOR    |  1   |               (Arthur et al. 2018)               |
|    LTB    |  1   |               (Chapuy et al. 2018)               |
|    LYN    |  1   |               (Chapuy et al. 2018)               |
|   MCL1    |  1   |               (Reddy et al. 2017)                |
|   MEF2B   |  1   |               (Morin et al. 2011)                |
|   MEF2C   |  1   |               (Arthur et al. 2018)               |
|    MGA    |  1   |               (Zhang et al. 2013)                |
|   MPEG1   |  1   |               (Morin et al. 2013)                |
|   MS4A1   |  1   |              (Rushton et al. 2020)               |
|   MTOR    |  1   |               (Zhang et al. 2013)                |
|    MYC    |  1   |           (L. Pasqualucci et al. 2001)           |
|   MYD88   |  1   |                (Ngo et al. 2011)                 |
|   MYOM2   |  1   |    (Laura Pasqualucci, Trifonov, et al. 2011)    |
|   NCOR2   |  1   |                                                  |
|   NFKB1   |  1   |                                                  |
|  NFKBIA   |  1   | (Lake et al. 2009; Russler-Germain et al. 2023)  |
|  NFKBIE   |  1   |               (Morin et al. 2016)                |
|  NFKBIZ   |  1   |               (Morin et al. 2016)                |
|   NLRC5   |  1   |                                                  |
|   NOL9    |  1   |              (Schmitz et al. 2018)               |
|  NOTCH1   |  1   |    (Laura Pasqualucci, Trifonov, et al. 2011)    |
|  NOTCH2   |  1   |               (Trøen et al. 2008)                |
|  OSBPL10  |  1   |               (Arthur et al. 2018)               |
|   P2RY8   |  1   |                (Lohr et al. 2012)                |
|   PCBP1   |  1   |                                                  |
|   PIM1    |  1   |           (L. Pasqualucci et al. 2001)           |
|   PIM2    |  1   |               (Arthur et al. 2018)               |
|  POU2AF1  |  1   |              (Krysiak et al. 2017)               |
|  POU2F2   |  1   |     (Zhang et al. 2013; Krysiak et al. 2017)     |
|   PRDM1   |  1   |  (**pasqualucciInactivationPRDM1BLIMP12006?**)   |
|   PRKDC   |  1   |  (Schmitz et al. 2018; Hübschmann et al. 2021)   |
|   PTEN    |  1   |                                                  |
|   PTPRD   |  1   |                                                  |
|    RB1    |  1   |               (Morin et al. 2013)                |
|   RFX7    |  1   |                                                  |
|   RFXAP   |  1   |                                                  |
|   RHOA    |  1   |           (L. Pasqualucci et al. 2001)           |
|   RRAGC   |  1   |               (Okosun et al. 2016)               |
|   S1PR2   |  1   |               (Morin et al. 2011)                |
|  SETD1B   |  1   |               (Reddy et al. 2017)                |
|   SF3B1   |  1   |                                                  |
|   SGK1    |  1   |               (Morin et al. 2011)                |
|   SIN3A   |  1   |                                                  |
|  SMARCA4  |  1   |     (Zhang et al. 2013; Krysiak et al. 2017)     |
|   SOCS1   |  1   |               (Morin et al. 2011)                |
|   SPEN    |  1   |               (Rossi et al. 2012)                |
|   STAT3   |  1   |               (Ohgami et al. 2014)               |
|   STAT6   |  1   |               (Yildiz et al. 2015)               |
|   TAF1    |  1   |               (Morin et al. 2013)                |
|  TBL1XR1  |  1   |             (Mareschal et al. 2016)              |
|   TET2    |  1   |            (Albuquerque et al. 2017)             |
|  TMEM30A  |  1   |               (Morin et al. 2011)                |
|  TMSB4X   |  1   |            (Albuquerque et al. 2017)             |
|  TNFAIP3  |  1   |              (Compagno et al. 2009)              |
| TNFRSF14  |  1   |               (Cheung et al. 2010)               |
|    TOX    |  1   |               (Reddy et al. 2017)                |
|   TP53    |  1   |               (Morin et al. 2011)                |
|  TRIP12   |  1   |                                                  |
|   TRRAP   |  1   |               (Parry et al. 2013)                |
|   UBE2A   |  1   |               (Reddy et al. 2017)                |
|   UNC5C   |  1   |                                                  |
|   USP7    |  1   |                                                  |
|  VPS13B   |  1   |                                                  |
|   WEE1    |  1   |              (Schmitz et al. 2018)               |
|   XPO1    |  1   |             (Mareschal et al. 2016)              |
|  ZC3H12A  |  1   |               (Arthur et al. 2018)               |
|  ZFP36L1  |  1   |               (Morin et al. 2011)                |
|  ZNF292   |  1   |               (Zhang et al. 2013)                |
|  ZNF608   |  1   |     (Zhang et al. 2013; Krysiak et al. 2017)     |

## Tier 2 DLBCL genes

|   Gene    | Tier |               Relevant references                |
|:---------:|:----:|:------------------------------------------------:|
|  ABI3BP   |  2   |               (Morin et al. 2013)                |
|  ADAMTS1  |  2   |             (Hübschmann et al. 2021)             |
|   AICDA   |  2   |               (Arthur et al. 2018)               |
|  ANKRD12  |  2   |             (Hübschmann et al. 2021)             |
|  ANKRD17  |  2   |               (Reddy et al. 2017)                |
|  ARID1B   |  2   |               (Reddy et al. 2017)                |
|  ARID5B   |  2   |               (Reddy et al. 2017)                |
|    ATR    |  2   |               (Reddy et al. 2017)                |
|   BIRC3   |  2   |               (Arthur et al. 2018)               |
|    BLK    |  2   |                                                  |
|  BRINP3   |  2   |               (Reddy et al. 2017)                |
|   BTBD3   |  2   |               (Reddy et al. 2017)                |
|  CADPS2   |  2   |             (Hübschmann et al. 2021)             |
|   CBLB    |  2   |               (Reddy et al. 2017)                |
|   CCL4    |  2   |               (Chapuy et al. 2018)               |
|   CD22    |  2   |               (Reddy et al. 2017)                |
|   CD44    |  2   |               (Arthur et al. 2018)               |
|   CD74    |  2   |               (Arthur et al. 2018)               |
|   CDC73   |  2   |               (Reddy et al. 2017)                |
|   CDH9    |  2   |               (Morin et al. 2013)                |
|   CHD1    |  2   |               (Reddy et al. 2017)                |
|   CHD8    |  2   |               (Reddy et al. 2017)                |
|   CHST2   |  2   |               (Reddy et al. 2017)                |
|   CNOT2   |  2   |             (Hübschmann et al. 2021)             |
|  CNTNAP5  |  2   |               (Morin et al. 2013)                |
|   COQ7    |  2   |               (Chapuy et al. 2018)               |
|   CRIP1   |  2   |               (Chapuy et al. 2018)               |
|   DCAF6   |  2   |               (Reddy et al. 2017)                |
|   DDX10   |  2   |               (Reddy et al. 2017)                |
|   DHX16   |  2   |             (Hübschmann et al. 2021)             |
|  DICER1   |  2   |               (Reddy et al. 2017)                |
|   DNAH5   |  2   |               (Morin et al. 2013)                |
|   DNM2    |  2   |             (Hübschmann et al. 2021)             |
|  DNMT3A   |  2   |               (Reddy et al. 2017)                |
|   DOCK1   |  2   |               (Chapuy et al. 2018)               |
|   DSG4    |  2   |               (Morin et al. 2013)                |
|  EIF2AK3  |  2   |                                                  |
|    EZR    |  2   |                                                  |
|  FAM102A  |  2   |                                                  |
|  FAM38B   |  2   |               (Morin et al. 2013)                |
|   FANK1   |  2   |                                                  |
|   FAT4    |  2   |               (Morin et al. 2013)                |
|   FCRL3   |  2   |                                                  |
|   FNBP1   |  2   |                                                  |
|   FNDC1   |  2   |               (Morin et al. 2013)                |
|   FOXP1   |  2   |               (Reddy et al. 2017)                |
|   FUBP1   |  2   |               (Reddy et al. 2017)                |
|   FUT5    |  2   |               (Chapuy et al. 2018)               |
|  GABRA2   |  2   |               (Chapuy et al. 2018)               |
|    GAK    |  2   |             (Hübschmann et al. 2021)             |
|   GNAS    |  2   |               (Reddy et al. 2017)                |
|  GOLGA5   |  2   |               (Reddy et al. 2017)                |
|   GPC5    |  2   |              (Schmitz et al. 2018)               |
|   GSG2    |  2   |              (Schmitz et al. 2018)               |
|   HDAC7   |  2   |               (Morin et al. 2013)                |
| HIST1H2AG |  2   |     (Morin et al. 2013; Krysiak et al. 2017)     |
|   HNF1B   |  2   |    (Laura Pasqualucci, Trifonov, et al. 2011)    |
|   HRAS    |  2   |               (Reddy et al. 2017)                |
|    ID3    |  2   |              (Schmitz et al. 2012)               |
|   IER2    |  2   |               (Morin et al. 2013)                |
|  IFNGR1   |  2   |               (Morin et al. 2013)                |
|   IGLL5   |  2   |          (Russler-Germain et al. 2023)           |
|   IKBKB   |  2   |               (Reddy et al. 2017)                |
|   IKBKE   |  2   |             (Hübschmann et al. 2021)             |
|    IL6    |  2   |               (Chapuy et al. 2018)               |
|   INO80   |  2   |               (Zhang et al. 2013)                |
|   IRAG2   |  2   |                                                  |
|   IRF1    |  2   |             (Hübschmann et al. 2021)             |
|   JAK1    |  2   |               (Zhang et al. 2013)                |
|   JAK3    |  2   |               (Zhang et al. 2013)                |
|   KCMF1   |  2   |               (Reddy et al. 2017)                |
|  KLHL21   |  2   |              (Schmitz et al. 2018)               |
|   LAMA5   |  2   |              (Schmitz et al. 2018)               |
|  LAPTM5   |  2   |             (Hübschmann et al. 2021)             |
|   LIN54   |  2   |               (Reddy et al. 2017)                |
|    LPP    |  2   |               (Arthur et al. 2018)               |
|   LRP12   |  2   |             (Hübschmann et al. 2021)             |
|   LRRN3   |  2   |               (Morin et al. 2013)                |
|   MAGT1   |  2   |               (Reddy et al. 2017)                |
|  MALAT1   |  2   |               (Arthur et al. 2018)               |
|  MAP2K1   |  2   |    (Shin et al. 2015; Louissaint et al. 2016)    |
|  MAP4K4   |  2   |               (Reddy et al. 2017)                |
|   MARK1   |  2   |               (Reddy et al. 2017)                |
|   MECOM   |  2   |               (Reddy et al. 2017)                |
|    MET    |  2   |               (Reddy et al. 2017)                |
|  MIR142   |  2   |              (Kwanhian et al. 2012)              |
| MIR155HG  |  2   |                                                  |
|   MPDZ    |  2   |               (Morin et al. 2013)                |
|   MSH2    |  2   |               (Reddy et al. 2017)                |
|   MSH6    |  2   |               (Reddy et al. 2017)                |
|    MYB    |  2   |               (Reddy et al. 2017)                |
|  MYBPC2   |  2   |              (Schmitz et al. 2018)               |
|   MYO1E   |  2   |                                                  |
|   N2RF2   |  2   |             (Hübschmann et al. 2021)             |
|   NANOG   |  2   |               (Chapuy et al. 2018)               |
|   NAV1    |  2   |               (Chapuy et al. 2018)               |
|   NCOA3   |  2   |                                                  |
|   NCOR1   |  2   |               (Reddy et al. 2017)                |
|   NEAT1   |  2   |               (Arthur et al. 2018)               |
|    NF1    |  2   |               (Reddy et al. 2017)                |
|   NFKB2   |  2   |               (Reddy et al. 2017)                |
|   NLRP5   |  2   |               (Morin et al. 2013)                |
|   NLRP8   |  2   |               (Chapuy et al. 2018)               |
|   ODZ3    |  2   |               (Morin et al. 2013)                |
|   P2RX5   |  2   |               (Morin et al. 2013)                |
|  PAPOLG   |  2   |              (Schmitz et al. 2018)               |
|   PASK    |  2   |               (Morin et al. 2013)                |
|   PAX5    |  2   |           (L. Pasqualucci et al. 2001)           |
|  PCDHB11  |  2   |               (Morin et al. 2013)                |
|   PCLO    |  2   |                (Lohr et al. 2012)                |
|  PDE4DIP  |  2   |               (Chapuy et al. 2018)               |
|   PDS5B   |  2   |   (Morin et al. 2013; Hübschmann et al. 2021)    |
|   PHF6    |  2   |               (Reddy et al. 2017)                |
|  PIK3CD   |  2   |               (Reddy et al. 2017)                |
|  PIK3R1   |  2   |               (Zhang et al. 2013)                |
|   PKD1    |  2   |               (Morin et al. 2013)                |
|   PNPO    |  2   |             (Hübschmann et al. 2021)             |
|   POGZ    |  2   |               (Morin et al. 2013)                |
|  PPP1R9B  |  2   |                                                  |
|   PRKCB   |  2   |               (Morin et al. 2013)                |
|   PRPS1   |  2   |               (Chapuy et al. 2018)               |
|   PTMA    |  2   |                                                  |
|   PTPN1   |  2   |                                                  |
|  PTPN23   |  2   |               (Morin et al. 2013)                |
|   PTPN6   |  2   |               (Reddy et al. 2017)                |
|   PTPRK   |  2   |               (Reddy et al. 2017)                |
|   RAC2    |  2   |             (Hübschmann et al. 2021)             |
|   RAD9A   |  2   |               (Chapuy et al. 2018)               |
|   RARA    |  2   |               (Reddy et al. 2017)                |
|    RCC    |  2   |                                                  |
|   RFTN1   |  2   |               (Arthur et al. 2018)               |
|   RHEX    |  2   |                                                  |
|   RHOH    |  2   |                                                  |
|  RUBCNL   |  2   |                                                  |
|   RUNX1   |  2   |               (Reddy et al. 2017)                |
|  SAMD9L   |  2   |               (Morin et al. 2013)                |
|   SARM1   |  2   |               (Morin et al. 2013)                |
|  SEL1L3   |  2   |                                                  |
|  SEPTIN9  |  2   |                                                  |
| SERPINA9  |  2   |               (Arthur et al. 2018)               |
|   SETD2   |  2   |               (Zhang et al. 2013)                |
|   SETD5   |  2   |               (Reddy et al. 2017)                |
|   SIAH2   |  2   |             (Hübschmann et al. 2021)             |
|  SLC34A2  |  2   |             (Hübschmann et al. 2021)             |
|   SMEK1   |  2   |               (Chapuy et al. 2018)               |
|   SRRM2   |  2   | (Morin et al. 2013; Russler-Germain et al. 2023) |
|  ST6GAL1  |  2   |               (Arthur et al. 2018)               |
|  STAT5B   |  2   |               (Zhang et al. 2013)                |
|    SYK    |  2   |               (Reddy et al. 2017)                |
|   SYPL1   |  2   |               (Morin et al. 2013)                |
|   TAP1    |  2   |              (Schmitz et al. 2018)               |
|  TBC1D4   |  2   |               (Arthur et al. 2018)               |
|   TCL1A   |  2   |                                                  |
|  TGFBR2   |  2   |               (Reddy et al. 2017)                |
|  TIPARP   |  2   |               (Reddy et al. 2017)                |
|   TLR2    |  2   |               (Chapuy et al. 2018)               |
|   TRAF3   |  2   |    (Laura Pasqualucci, Trifonov, et al. 2011)    |
|   TRAF6   |  2   |             (Hübschmann et al. 2021)             |
|  UBE2J1   |  2   |                                                  |
|   UBR5    |  2   |               (Zhang et al. 2013)                |
|   UNC5B   |  2   |             (Hübschmann et al. 2021)             |
|   UNC5D   |  2   |                                                  |
|    WAC    |  2   |               (Reddy et al. 2017)                |
|   WDFY3   |  2   |               (Morin et al. 2013)                |
|   WNK1    |  2   |             (Hübschmann et al. 2021)             |
|   XBP1    |  2   |                                                  |
|    YY1    |  2   |               (Reddy et al. 2017)                |
|  ZBTB7A   |  2   |               (Reddy et al. 2017)                |
|  ZCCHC7   |  2   |               (Arthur et al. 2018)               |
|   ZEB2    |  2   |               (Zhang et al. 2013)                |
|   ZFAT    |  2   |               (Reddy et al. 2017)                |
|    ZFX    |  2   |               (Reddy et al. 2017)                |
|  ZNF217   |  2   |             (Hübschmann et al. 2021)             |
|  ZNF423   |  2   |               (Chapuy et al. 2018)               |

# References

<div id="refs" class="references csl-bib-body hanging-indent">

<div id="ref-albuquerqueEnhancingKnowledgeDiscovery2017a"
class="csl-entry">

Albuquerque, Marco A., Bruno M. Grande, Elie J. Ritch, Prasath
Pararajalingam, Selin Jessa, Martin Krzywinski, Jasleen K. Grewal,
Sohrab P. Shah, Paul C. Boutros, and Ryan D. Morin. 2017. “Enhancing
Knowledge Discovery from Cancer Genomics Data with Galaxy.”
*GigaScience* 6 (5): 1–13. <https://doi.org/10.1093/gigascience/gix015>.

</div>

<div id="ref-arthurGenomewideDiscoverySomatic2018" class="csl-entry">

Arthur, Sarah E., Aixiang Jiang, Bruno M. Grande, Miguel Alcaide, Razvan
Cojocaru, Christopher K. Rushton, Anja Mottok, et al. 2018. “Genome-Wide
Discovery of Somatic Regulatory Variants in Diffuse Large <span
class="nocase">B-cell</span> Lymphoma.” *Nature Communications* 9 (1):
4001. <https://doi.org/10.1038/s41467-018-06354-3>.

</div>

<div id="ref-bohleRoleEarlyBcell2013" class="csl-entry">

Bohle, V., C. Döring, M.-L. Hansmann, and R. Küppers. 2013. “Role of
Early <span class="nocase">B-cell</span> Factor 1 (EBF1) in Hodgkin
Lymphoma.” *Leukemia* 27 (3): 671–79.
<https://doi.org/10.1038/leu.2012.280>.

</div>

<div id="ref-chapuyMolecularSubtypesDiffuse2018b" class="csl-entry">

Chapuy, Bjoern, Chip Stewart, Andrew J. Dunford, Jaegil Kim, Atanas
Kamburov, Robert A. Redd, Mike S. Lawrence, et al. 2018. “Molecular
Subtypes of Diffuse Large B Cell Lymphoma Are Associated with Distinct
Pathogenic Mechanisms and Outcomes.” *Nature Medicine* 24 (5): 679–90.
<https://doi.org/10.1038/s41591-018-0016-8>.

</div>

<div id="ref-cheungAcquiredTNFRSF14Mutations2010a" class="csl-entry">

Cheung, K.-John J., Nathalie A. Johnson, Joslynn G. Affleck, Tesa
Severson, Christian Steidl, Susana Ben-Neriah, Jacqueline Schein, et al.
2010. “Acquired TNFRSF14 Mutations in Follicular Lymphoma Are Associated
with Worse Prognosis.” *Cancer Research* 70 (22): 9166–74.
<https://doi.org/10.1158/0008-5472.CAN-10-2460>.

</div>

<div id="ref-compagnoMutationsMultipleGenes2009a" class="csl-entry">

Compagno, Mara, Wei Keat Lim, Adina Grunn, Subhadra V. Nandula, Manisha
Brahmachary, Qiong Shen, Francesco Bertoni, et al. 2009. “Mutations of
Multiple Genes Cause Deregulation of <span
class="nocase">NF-kappaB</span> in Diffuse Large <span
class="nocase">B-cell</span> Lymphoma.” *Nature* 459 (7247): 717–21.
<https://doi.org/10.1038/nature07968>.

</div>

<div id="ref-dunsCharacterizationDLBCLPMBL2021b" class="csl-entry">

Duns, Gerben, Elena Viganò, Daisuke Ennishi, Clementine Sarkozy, Stacy
S. Hung, Elizabeth Chavez, Katsuyoshi Takata, et al. 2021.
“Characterization of DLBCL with a PMBL Gene Expression Signature.”
*Blood* 138 (2): 136–48. <https://doi.org/10.1182/blood.2020007683>.

</div>

<div id="ref-hubschmannMutationalMechanismsShaping2021b"
class="csl-entry">

Hübschmann, Daniel, Kortine Kleinheinz, Rabea Wagener, Stephan H.
Bernhart, Cristina López, Umut H. Toprak, Stephanie Sungalee, et al.
2021. “Mutational Mechanisms Shaping the Coding and Noncoding Genome of
Germinal Center Derived <span class="nocase">B-cell</span> Lymphomas.”
*Leukemia* 35 (7): 2002–16.
<https://doi.org/10.1038/s41375-021-01251-z>.

</div>

<div id="ref-khodabakhshiRecurrentTargetsAberrant2012"
class="csl-entry">

Khodabakhshi, Alireza Hadj, Ryan D. Morin, Anthony P. Fejes, Andrew J.
Mungall, Karen L. Mungall, Madison Bolger-Munro, Nathalie A. Johnson, et
al. 2012. “Recurrent Targets of Aberrant Somatic Hypermutation in
Lymphoma.” *Oncotarget* 3 (11): 1308–19.
<https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3717795/>.

</div>

<div id="ref-krysiakRecurrentSomaticMutations2017b" class="csl-entry">

Krysiak, Kilannin, Felicia Gomez, Brian S. White, Matthew Matlock,
Christopher A. Miller, Lee Trani, Catrina C. Fronick, et al. 2017.
“Recurrent Somatic Mutations Affecting <span
class="nocase">B-cell</span> Receptor Signaling Pathway Genes in
Follicular Lymphoma.” *Blood* 129 (4): 473–83.
<https://doi.org/10.1182/blood-2016-07-729954>.

</div>

<div id="ref-kwanhianMicroRNA142Mutated202012b" class="csl-entry">

Kwanhian, Wiyada, Dido Lenze, Julia Alles, Natalie Motsch, Stephanie
Barth, Celina Döll, Jochen Imig, et al. 2012. “MicroRNA-142 Is Mutated
in about 20% of Diffuse Large <span class="nocase">B-cell</span>
Lymphoma.” *Cancer Medicine* 1 (2): 141–55.
<https://doi.org/10.1002/cam4.29>.

</div>

<div id="ref-lakeMutationsNFKBIAEncoding2009" class="csl-entry">

Lake, Annette, Lesley A. Shield, Pablo Cordano, Daniel T. Y. Chui, Julie
Osborne, Shauna Crae, Katherine S. Wilson, et al. 2009. “Mutations of
NFKBIA, Encoding IkappaB Alpha, Are a Recurrent Finding in Classical
Hodgkin Lymphoma but Are Not a Unifying Feature of Non-<span
class="nocase">EBV-associated</span> Cases.” *International Journal of
Cancer* 125 (6): 1334–42. <https://doi.org/10.1002/ijc.24502>.

</div>

<div id="ref-lenzOncogenicCARD11Mutations2008" class="csl-entry">

Lenz, Georg, R Eric Davis, Vu N Ngo, Lloyd Lam, Thaddeus C George,
George W Wright, Sandeep S Dave, et al. 2008. “Oncogenic CARD11
Mutations in Human Diffuse Large B Cell Lymphoma.” *Science* 319 (5870):
1676–79.

</div>

<div id="ref-lohrDiscoveryPrioritizationSomatic2012a" class="csl-entry">

Lohr, Jens G., Petar Stojanov, Michael S. Lawrence, Daniel Auclair,
Bjoern Chapuy, Carrie Sougnez, Peter Cruz-Gordillo, et al. 2012.
“Discovery and Prioritization of Somatic Mutations in Diffuse Large
<span class="nocase">B-cell</span> Lymphoma (DLBCL) by Whole-Exome
Sequencing.” *Proceedings of the National Academy of Sciences of the
United States of America* 109 (10): 3879–84.
<https://doi.org/10.1073/pnas.1121343109>.

</div>

<div id="ref-louissaintPediatrictypeNodalFollicular2016a"
class="csl-entry">

Louissaint, Abner, Kristian T. Schafernak, Julia T. Geyer, Alexandra E.
Kovach, Mahmoud Ghandi, Dita Gratzinger, Christine G. Roth, et al. 2016.
“Pediatric-Type Nodal Follicular Lymphoma: A Biologically Distinct
Lymphoma with Frequent MAPK Pathway Mutations.” *Blood* 128 (8):
1093–1100. <https://doi.org/10.1182/blood-2015-12-682591>.

</div>

<div id="ref-mareschalWholeExomeSequencing2016" class="csl-entry">

Mareschal, Sylvain, Sydney Dubois, Pierre-Julien Viailly, Philippe
Bertrand, Elodie Bohers, Catherine Maingonnat, Jean-Philippe Jaïs, et
al. 2016. “Whole Exome Sequencing of Relapsed/Refractory Patients
Expands the Repertoire of Somatic Mutations in Diffuse Large <span
class="nocase">B-cell</span> Lymphoma.” *Genes, Chromosomes & Cancer* 55
(3): 251–67. <https://doi.org/10.1002/gcc.22328>.

</div>

<div id="ref-morinGeneticLandscapesRelapsed2016" class="csl-entry">

Morin, Ryan D., Sarit Assouline, Miguel Alcaide, Arezoo Mohajeri,
Rebecca L. Johnston, Lauren Chong, Jasleen Grewal, et al. 2016. “Genetic
Landscapes of Relapsed and Refractory Diffuse Large B-Cell Lymphomas.”
*Clinical Cancer Research: An Official Journal of the American
Association for Cancer Research* 22 (9): 2290–300.
<https://doi.org/10.1158/1078-0432.CCR-15-2123>.

</div>

<div id="ref-morinSomaticMutationsAltering2010a" class="csl-entry">

Morin, Ryan D., Nathalie A. Johnson, Tesa M. Severson, Andrew J.
Mungall, Jianghong An, Rodrigo Goya, Jessica E. Paul, et al. 2010.
“Somatic Mutations Altering EZH2 (Tyr641) in Follicular and Diffuse
Large <span class="nocase">B-cell</span> Lymphomas of Germinal-Center
Origin.” *Nature Genetics* 42 (2): 181–85.
<https://doi.org/10.1038/ng.518>.

</div>

<div id="ref-morinFrequentMutationHistonemodifying2011"
class="csl-entry">

Morin, Ryan D., Maria Mendez-Lago, Andrew J. Mungall, Rodrigo Goya,
Karen L. Mungall, Richard D. Corbett, Nathalie A. Johnson, et al. 2011.
“Frequent Mutation of Histone-Modifying Genes in Non-Hodgkin Lymphoma.”
*Nature* 476 (7360): 298–303. <https://doi.org/10.1038/nature10351>.

</div>

<div id="ref-morinMutationalStructuralAnalysis2013" class="csl-entry">

Morin, Ryan D., Karen Mungall, Erin Pleasance, Andrew J. Mungall,
Rodrigo Goya, Ryan D. Huff, David W. Scott, et al. 2013. “Mutational and
Structural Analysis of Diffuse Large <span class="nocase">B-cell</span>
Lymphoma Using Whole-Genome Sequencing.” *Blood* 122 (7): 1256–65.
<https://doi.org/10.1182/blood-2013-02-483727>.

</div>

<div id="ref-ngoOncogenicallyActiveMYD882011a" class="csl-entry">

Ngo, Vu N., Ryan M. Young, Roland Schmitz, Sameer Jhavar, Wenming Xiao,
Kian-Huat Lim, Holger Kohlhammer, et al. 2011. “Oncogenically Active
MYD88 Mutations in Human Lymphoma.” *Nature* 470 (7332): 115–19.
<https://doi.org/10.1038/nature09671>.

</div>

<div id="ref-ohgamiSTAT3MutationsAre2014" class="csl-entry">

Ohgami, Robert S., Lisa Ma, Ahmad Monabati, James L. Zehnder, and Daniel
A. Arber. 2014. “STAT3 Mutations Are Present in Aggressive <span
class="nocase">B-cell</span> Lymphomas Including a Subset of Diffuse
Large <span class="nocase">B-cell</span> Lymphomas with CD30
Expression.” *Haematologica* 99 (7): e105–107.
<https://doi.org/10.3324/haematol.2013.101543>.

</div>

<div id="ref-okosunRecurrentMTORC1activatingRRAGC2016a"
class="csl-entry">

Okosun, Jessica, Rachel L. Wolfson, Jun Wang, Shamzah Araf, Lucy
Wilkins, Brian M. Castellano, Leire Escudero-Ibarz, et al. 2016.
“Recurrent <span class="nocase">mTORC1-activating RRAGC</span> Mutations
in Follicular Lymphoma.” *Nature Genetics* 48 (2): 183–88.
<https://doi.org/10.1038/ng.3473>.

</div>

<div id="ref-pararajalingamCodingNoncodingDrivers2020"
class="csl-entry">

Pararajalingam, Prasath, Krysta M. Coyle, Sarah E. Arthur, Nicole
Thomas, Miguel Alcaide, Barbara Meissner, Merrill Boyle, et al. 2020.
“Coding and Noncoding Drivers of Mantle Cell Lymphoma Identified Through
Exome and Genome Sequencing.” *Blood* 136 (5): 572–84.
<https://doi.org/10.1182/blood.2019002385>.

</div>

<div id="ref-parryWholeExomeSequencing2013" class="csl-entry">

Parry, Marina, Matthew J. J. Rose-Zerilli, Jane Gibson, Sarah Ennis,
Renata Walewska, Jade Forster, Helen Parker, et al. 2013. “Whole Exome
Sequencing Identifies Novel Recurrently Mutated Genes in Patients with
Splenic Marginal Zone Lymphoma.” *PloS One* 8 (12): e83244.
<https://doi.org/10.1371/journal.pone.0083244>.

</div>

<div id="ref-pasqualucciInactivatingMutationsAcetyltransferase2011a"
class="csl-entry">

Pasqualucci, Laura, David Dominguez-Sola, Annalisa Chiarenza, Giulia
Fabbri, Adina Grunn, Vladimir Trifonov, Lawryn H. Kasper, et al. 2011.
“Inactivating Mutations of Acetyltransferase Genes in <span
class="nocase">B-cell</span> Lymphoma.” *Nature* 471 (7337): 189–95.
<https://doi.org/10.1038/nature09730>.

</div>

<div id="ref-pasqualucciAnalysisCodingGenome2011" class="csl-entry">

Pasqualucci, Laura, Vladimir Trifonov, Giulia Fabbri, Jing Ma, Davide
Rossi, Annalisa Chiarenza, Victoria A. Wells, et al. 2011. “Analysis of
the Coding Genome of Diffuse Large <span class="nocase">B-cell</span>
Lymphoma.” *Nature Genetics* 43 (9): 830–37.
<https://doi.org/10.1038/ng.892>.

</div>

<div id="ref-pasqualucciHypermutationMultipleProtooncogenes2001a"
class="csl-entry">

Pasqualucci, L., P. Neumeister, T. Goossens, G. Nanjangud, R. S.
Chaganti, R. Küppers, and R. Dalla-Favera. 2001. “Hypermutation of
Multiple Proto-Oncogenes in <span class="nocase">B-cell</span> Diffuse
Large-Cell Lymphomas.” *Nature* 412 (6844): 341–46.
<https://doi.org/10.1038/35085588>.

</div>

<div id="ref-reddyGeneticFunctionalDrivers2017" class="csl-entry">

Reddy, Anupama, Jenny Zhang, Nicholas S Davis, Andrea B Moffitt,
Cassandra L Love, Alexander Waldrop, Sirpa Leppa, et al. 2017. “Genetic
and Functional Drivers of Diffuse Large B Cell Lymphoma.” *Cell* 171
(2): 481–494.e15.

</div>

<div id="ref-rossiCodingGenomeSplenic2012c" class="csl-entry">

Rossi, Davide, Vladimir Trifonov, Marco Fangazio, Alessio Bruscaggin,
Silvia Rasi, Valeria Spina, Sara Monti, et al. 2012. “The Coding Genome
of Splenic Marginal Zone Lymphoma: Activation of NOTCH2 and Other
Pathways Regulating Marginal Zone Development.” *The Journal of
Experimental Medicine* 209 (9): 1537–51.
<https://doi.org/10.1084/jem.20120904>.

</div>

<div id="ref-rushtonGeneticEvolutionaryPatterns2020" class="csl-entry">

Rushton, Christopher K., Sarah E. Arthur, Miguel Alcaide, Matthew
Cheung, Aixiang Jiang, Krysta M. Coyle, Kirstie L. S. Cleary, et al.
2020. “Genetic and Evolutionary Patterns of Treatment Resistance in
Relapsed <span class="nocase">B-cell</span> Lymphoma.” *Blood Advances*
4 (13): 2886–98. <https://doi.org/10.1182/bloodadvances.2020001696>.

</div>

<div id="ref-russler-germainMutationsAssociatedProgression2023b"
class="csl-entry">

Russler-Germain, David A., Kilannin Krysiak, Cody A. Ramirez, Matthew
Mosior, Marcus P. Watkins, Felicia Gomez, Zachary L. Skidmore, et al.
2023. “Mutations Associated with Progression in Follicular Lymphoma
Predict Inferior Outcomes at Diagnosis: Alliance A151303.” *Blood
Advances* 7: 5524–39.
<https://doi.org/10.1182/bloodadvances.2023010779>.

</div>

<div id="ref-schmitzGeneticsPathogenesisDiffuse2018a" class="csl-entry">

Schmitz, Roland, George W. Wright, Da Wei Huang, Calvin A. Johnson,
James D. Phelan, James Q. Wang, Sandrine Roulland, et al. 2018.
“Genetics and Pathogenesis of Diffuse Large B-Cell Lymphoma.” *The New
England Journal of Medicine* 378 (15): 1396–1407.
<https://doi.org/10.1056/NEJMoa1801445>.

</div>

<div id="ref-schmitzBurkittLymphomaPathogenesis2012" class="csl-entry">

Schmitz, Roland, Ryan M. Young, Michele Ceribelli, Sameer Jhavar,
Wenming Xiao, Meili Zhang, George Wright, et al. 2012. “Burkitt Lymphoma
Pathogenesis and Therapeutic Targets from Structural and Functional
Genomics.” *Nature* 490 (7418): 116–20.
<https://doi.org/10.1038/nature11378>.

</div>

<div id="ref-schollMutationsRegionFAS2007" class="csl-entry">

Scholl, Vanesa, Claudio Gustavo Stefanoff, Rocio Hassan, Nelson Spector,
and Ilana Zalcberg Renault. 2007. “Mutations Within the 5’ Region of
FAS/CD95 Gene in Nodal Diffuse Large <span class="nocase">B-cell</span>
Lymphoma.” *Leukemia & Lymphoma* 48 (5): 957–63.
<https://doi.org/10.1080/10428190701230858>.

</div>

<div id="ref-shinBRAFV600EMAP2K12015" class="csl-entry">

Shin, Sang-Yong, Seung-Tae Lee, Hee-Jin Kim, Chang-Seok Ki, Chul Won
Jung, Jong-Won Kim, and Sun-Hee Kim. 2015. “BRAF V600E and MAP2K1
Mutations in Hairy Cell Leukemia and Splenic Marginal Zone Lymphoma
Cases.” *Annals of Laboratory Medicine* 35 (2): 257–59.
<https://doi.org/10.3343/alm.2015.35.2.257>.

</div>

<div id="ref-tanakaFrequentIncidenceSomatic1992" class="csl-entry">

Tanaka, S., D. C. Louie, J. A. Kant, and J. C. Reed. 1992. “Frequent
Incidence of Somatic Mutations in Translocated BCL2 Oncogenes of
Non-Hodgkin’s Lymphomas.” *Blood* 79 (1): 229–37.

</div>

<div id="ref-tiacciBRAFMutationsHairycell2011a" class="csl-entry">

Tiacci, Enrico, Vladimir Trifonov, Gianluca Schiavoni, Antony Holmes,
Wolfgang Kern, Maria Paola Martelli, Alessandra Pucciarini, et al. 2011.
“BRAF Mutations in Hairy-Cell Leukemia.” *The New England Journal of
Medicine* 364 (24): 2305–15. <https://doi.org/10.1056/NEJMoa1014209>.

</div>

<div id="ref-troenNOTCH2MutationsMarginal2008" class="csl-entry">

Trøen, Gunhild, Iwona Wlodarska, Abdirashid Warsame, Silvia Hernández
Llodrà, Christiane De Wolf-Peeters, and Jan Delabie. 2008. “NOTCH2
Mutations in Marginal Zone Lymphoma.” *Haematologica* 93 (7): 1107–9.
<https://doi.org/10.3324/haematol.11635>.

</div>

<div id="ref-yildizActivatingSTAT6Mutations2015c" class="csl-entry">

Yildiz, Mehmet, Hongxiu Li, Denzil Bernard, Nisar A. Amin, Peter
Ouillette, Siân Jones, Kamlai Saiya-Cork, et al. 2015. “Activating STAT6
Mutations in Follicular Lymphoma.” *Blood* 125 (4): 668–79.
<https://doi.org/10.1182/blood-2014-06-582650>.

</div>

<div id="ref-zhangGeneticHeterogeneityDiffuse2013" class="csl-entry">

Zhang, Jenny, Vladimir Grubor, Cassandra L Love, Anjishnu Banerjee,
Kristy L Richards, Piotr A Mieczkowski, Cherie Dunphy, et al. 2013.
“Genetic Heterogeneity of Diffuse Large <span
class="nocase">B-cell</span> Lymphoma.” January.

</div>

</div>
