---
bibliography: 'morinlab.bib'
csl: 'NLM.csl'
link-citations: true
nocite: |
  @gomezUltraDeepSequencingReveals2023, 
---
[[_TOC_]]

## Overview
Mutations in the HLA-C gene have been associated with a loss of cell surface expression of HLA class I molecules, which are essential for presenting tumor antigens to cytotoxic T cells. This is a common mechanism of immune escape in DLBCL. Deletions of this gene are more commonly reported than HLA-C mutations.
The mutation pattern in DLBCL implies the preferential accumulation of *inactivating mutations*. Different analytical strategies relating to the mapping of sequencing data and subtracting common germline variants can complicate the detection of mutations in this and other HLA genes. Likely owing to this, the rate of mutations is highly variable across studies and the true mutation rate has not been firmly established. 



## Relevance tier by entity

|Entity|Tier|Description               |
|:------:|:----:|--------------------------|
|![PMBL](images/icons/PMBL_tier2.png)|2|relevance in PMBL/cHL/GZL not firmly established[@gomezUltraDeepSequencingReveals2023]|
|![DLBCL](images/icons/DLBCL_tier1.png) |1   |high-confidence DLBCL gene|

## Mutation incidence in large patient cohorts (GAMBL reanalysis)

[[include:DLBCL_HLA-C.md]]

## Mutation pattern and selective pressure estimates

[[include:dnds_HLA-C.md]]

[[include:browser_HLA-C.md]]

## Expression
![](images/gene_expression/HLA-C_by_pathology.svg)
<!-- ORIGIN: gomezUltraDeepSequencingReveals2023 -->
<!-- PMBL: gomezUltraDeepSequencingReveals2023 -->

[[include:mermaid_HLA-C.md]]

## References

